psi4/driver/driver_cbs.py

```diff
@@ -1681,6 +1488,7 @@ def cbs(func, label, **kwargs):
             finalquantity.print_out()
     elif ptype == 'hessian':
         finalquantity = finalhessian
+        wfn.set_gradient(finalgradient)
```

Comment:
If there is a good reason for not doing this with all `frequency(cbs, ...)` calculations, let me know - otherwise the frequency calculations fail (as the gradient is not set properly).

Reply:
good addition